src/hooks/useAddressHistory.ts

```diff
@@ -0,0 +1,77 @@
+import { atom, useRecoilState } from "recoil";
+import { useAddressInput } from "hooks/useAddressInput";
+import { useNavigate } from "react-router-dom";
+import { isValidAddress } from "utils";
+import useNotification from "hooks/useNotification";
+import { ROUTES } from "consts";
+import { recoilPersist } from "recoil-persist";
+import { Address } from "ton";
+import { useEffect } from "react";
+import { useJettonAddress } from "hooks/useJettonAddress";
+
+const { persistAtom } = recoilPersist();
```

Comment:
if someone else uses this default key, would it override the address history? if so, need to provide a different key here.